Source/carthage/Bootstrap.swift

```diff
@@ -26,12 +26,35 @@ public struct BootstrapCommand: CommandType {
 					carthage.println(formatting.bullets + "No Cartfile.resolved found, updating dependencies")
 					return project.updateDependencies(shouldCheckout: options.checkoutAfterUpdate)
 				}
-
+				
+				let checkDependencies: SignalProducer<(), CarthageError>
+				if let depsToUpdate = options.dependenciesToUpdate {
+					checkDependencies =
+						project
+						.loadResolvedCartfile()
+						.flatMap(.Concat) { resolvedCartfile -> SignalProducer<(), CarthageError> in
+							let resolvedDependencyNames = resolvedCartfile.dependencies.map { $0.project.name }
+							let unresolvedDependencyNames = Set(depsToUpdate).subtract(resolvedDependencyNames)
+							
+							if !unresolvedDependencyNames.isEmpty {
+								return SignalProducer<(), CarthageError>(error:.UnresolvedDependencies([String](unresolvedDependencyNames).sort()))
+							}
+							
+							return .empty
+						}
+					}
+					else {
+						checkDependencies = .empty
+					}
```

Comment:
The indentation of these last 4 lines off. They should be at the same level as the `if`.